src/api/participantsRouter.ts

```diff
@@ -28,10 +31,29 @@ const idParser = z.object({
   participantId: z.string(),
 });
 
-participantsRouter.get('/:participantId/admin', async (req, res) => {
-  const { participantId } = idParser.parse(req.params);
-  const adminUsers = await User.query()
-    .where('role', 'admin')
-    .where('participantId', participantId);
-  res.status(200).json(adminUsers);
+const invitationParser = z.object({
+  firstName: z.string(),
+  lastName: z.string(),
+  email: z.string(),
+  jobFunction: z.nativeEnum(UserRole),
+});
+
+participantsRouter.post('/:participantId/invite', async (req, res) => {
+  try {
```

Comment:
Do we have enough set up to check that the signed-in user belongs to that participant?